### ISSUE TITLE
E492: Not an editor command

### ISSUE THREAD

**Issue by mirenradia** [alker0/chezmoi.vim#70]
When I try and use this plugin on a system with Vim 8.0 (running Rocky Linux 8 so this is the latest version) and do
```
chezmoi edit <some chezmoi managed file>
```
I get the following errors pop up
```
Error detected while processing BufWinEnter Auto commands for "<buffer=1>":
E492: Not an editor command: ++once autocmd! chezmoi_filetypedetect FileType <buffer>
Error detected while processing VimEnter Auto commands for "<buffer=1>":
E492: Not an editor command: ++once autocmd! chezmoi_filetypedetect FileType <buffer>
```
I also get similar errors when entering the command line.

I load this plugin using [Vundle](https://github.com/VundleVim/Vundle.vim) and my `.vimrc` can be found [here](https://github.com/mirenradia/dotfiles/blob/main/dot_vimrc.tmpl).

Any ideas? Is this version of Vim too old to support this plugin?

**Comment by alker0** [comment]
Hi. Thank you for reporting it.

The error comes from -- as you can see the error message -- using the `++once` argument of the `autocmd` command that doesn't support Vim 8.0. I made this plugin use it because I didn't know the feature was that new. The error should be fixed via the PR #71.

Please update this plugin and check if the issue is solved.

**Comment by mirenradia** [comment]
This has solved my problem. Thanks for fixing it promptly!
